app/services/api/likeco-api.ts

```diff
@@ -354,4 +354,54 @@ export class LikeCoAPI {
       return { kind: "bad-data" }
     }
   }
+  
+  /**
+   * Fetch Civic Liker staking info
+   */
+  async fetchCivicLikerStakingInfo(): Promise<Types.CivicLikerStakingInfoResult> {
+    const response: ApiResponse<Types.CivicLikerStakingInfo> = await this.apisauce.get('/civic/staking/info')
+
+    if (!response.ok) {
+      const problem = getGeneralApiProblem(response)
+      if (problem) return problem
+    }
+
+    try {
+      const { operatorAddress, stakingAmountTarget } = response.data
+      return {
+        kind: "ok",
+        data: {
+          operatorAddress,
+          stakingAmountTarget,
+        },
+      }
+    } catch {
+      return { kind: "bad-data" }
+    }
+  }
+
+  /**
+   * Fetch Civic Liker staking
+   */
+  async fetchCivicLikerStakingByAddress(address: string): Promise<Types.CivicLikerStakingResult> {
+    const response: ApiResponse<Types.CivicLikerStaking> = await this.apisauce.get(`/civic/staking?address=${address}`)
```

Comment:
Does this mean no one will ever use the auth-ed version of this api in the future?

Reply:
Yes